### ISSUE TITLE
Please allow nuget installation

### ISSUE THREAD

**Issue by yawkale** [ccasalicchio/UmbracoPackages#1]

**Comment by ccasalicchio** [comment]
I'll look into it!

**Comment by wiedikerli** [comment]
@ccasalicchio any update on this one?

**Comment by ccasalicchio** [comment]
Sorry guys, I've been slammed with other work. But I plan to get this done in the next while...

**Comment by ReVoid** [comment]
Really usefull package, I hope you did't thown it away :) Still waiting for nuget

**Comment by ccasalicchio** [comment]
Hey Dmitriy... I sure haven't ... just haven't had time to revisit the code.

Thank you for reminding me

Best wishes
Carlos Casalicchio
[image: Skype: casalicchio] <http://puul.ga/n4iv9bnh>
<http://puul.ga/6uzcQsmW>   <http://puul.ga/h9KyWTfu>
<http://puul.ga/g89Tsdxq>   <http://puul.ga/4R7oViGj>
<http://puul.ga/8tlkcrxi>   <http://puul.ga/bziyyf37>
<http://puul.ga/YcbmDf4s>   <http://puul.ga/udVmFUV8>
<http://puul.ga/iTSVtpg3>   <http://puul.ga/sx2GkN7N>
<http://puul.ga/ifkhhyuq>   <http://puul.ga/mnfs1tfl>
<http://puul.ga/jrg1rkh7>

    "A luta pela vida, nem sempre é vantajosa aos fortes, nem aos espertos
    Mais cedo ou mais tarde, quem cativa a vitória é aquele que crê plenamente:
    'Eu conseguirei'" -Napoleon Hill

**Comment by ReVoid** [comment]
@yawkale Today I user your package again, It works with Umbraco 7.15.3

Still waiting for nuget installation :)

**Comment by ccasalicchio** [comment]
Thanks for the update. I was actually considering if I should still bother with version 7, or migration to version 8... so thanks for the feedback

**Comment by ReVoid** [comment]
Your welcome. I think Umbraco 7 will live for several years, so it would be good to make the package maintainable.
By the way, I didn't test it with v8, may be it still works :)

**Comment by simkinsws** [comment]
Hey does it works with v8/9/10?

**Comment by ccasalicchio** [comment]
It does not, but I'll update it soon

**Comment by simkinsws** [comment]
> 

Can I some how take your service on my own umbraco ? I need help with something and cannot fix it by my self maybe you could give an advice or even take a payment to help .. thanks and I like your repos!

**Comment by ccasalicchio** [comment]
yes that works, what help do you need? you can message me on discord Lyt0sBR#4333

**Comment by simkinsws** [comment]
Well cannot find you on splatdev.com there is another way to get you? Maybe discord or some how ?

**Comment by ccasalicchio** [comment]
Yes , you can find me on discord Lyt0sBR#4333

**Comment by ccasalicchio** [comment]
I've started creating the nuget packages for these plugins. I hope it helps!

**Comment by ccasalicchio** [comment]
nuget packages now available!
